src/main/java/net/dv8tion/jda/core/audit/AuditLogKey.java

```diff
@@ -153,6 +155,18 @@
      */
     CHANNEL_TOPIC("topic"),
 
+    /**
+     * Change of the {@link net.dv8tion.jda.core.entities.TextChannel#getSlowmode() TextChannel.getSlowmode()} value.
+     * <br>Only for {@link net.dv8tion.jda.core.entities.ChannelType#TEXT ChannelType.Text}
+     *
+     * <p>Expected type: <b>Integer</b>
+     *
+     * @incubating
+     *         Slowmode is currently only available through the API and might still be changed in the future.
+     */
+    @Incubating
```

Comment:
Since we will be merging this on deploy we probably don't need to mark this as incubating